bats/tzbtc.bats

```diff
@@ -37,7 +37,7 @@ setup () {
 
 @test "invoking tzbtc 'printInitialStorage' command" {
   result="$($tzbtc printInitialStorage --owner-address tz1f1S7V2hZJ3mhj47djb5j1saek8c2yB2Cx)"
-  [ "$result" == '{ { Elt 0x0501000000056f776e6572 0x050a000000160000d476acd953eb55d38c398c85c3f53e19b62b167a }; { CDR; NIL operation; PAIR }; 0; False }' ]
+  [ "$result" == 'Pair { Elt 0x0501000000056f776e6572 0x050a000000160000d476acd953eb55d38c398c85c3f53e19b62b167a } (Pair { CDR; NIL operation; PAIR } (Pair 0 False))' ]
```

Comment:
Uh-oh, do you know why this was needed?

AFAIK either syntax used here should work :thinking:

Reply:
We don't linearize untyped right combs any more, as in general that's not quite correct, and typed value is printed unoptimized by default. I could print an optimized value instead, although it's a bit involved with Lorentz values (as is the case here)

Reply:
I've pushed 7ae9e983e058b5a25faad3ea3936bd36b73389ac amending this.

Reply:
got it, thanks for the explanation :ok_hand: 

Let's keep the version with the optimization then, if you don't mind